schemas/oic.sec.ace2.json

```diff
@@ -1,11 +1,18 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.sec.ace2.json#",
-  "title": "Subject based Access Control Entry (ACE) object definition",
+  "title": "Subject-based Access Control Entry (ACE) object definition",
   "definitions": {
     "oic.sec.ace2": {
       "type": "object",
       "properties": {
+        "aceid": {
```

Comment:
As discussed, aceid can't be required due to the 1485 text that describes cases where aceid is omitted:

> a.	If an UPDATE representation includes the “aces” array Property, then:
> . . .
>> ii.	Supplied ACEs without an “aceid” SHALL be appended to the existing “aces” array, and a unique (to the acl2 Resource) “aceid” SHALL be created and assigned to the new ACE by the Server.  The “aceid” of a deleted ACE should not be reused, to improve the determinism of the interface and reduce opportunity for race conditions.

I just forgot about this case when I made the comment above (I chalk it up as "brain damage"...)